sycl/test-e2e/Matrix/element_wise_all_ops_impl.hpp

```diff
@@ -105,8 +106,11 @@ void verify_op_c(const T l, const T r, const float ref, OP op) {
 
 // Avoid same kernel name for different types
 template <typename T, class name> class ewops_a {};
-template <typename T, size_t NROWS, size_t NCOLS, size_t SROWS, size_t SCOLS>
-void test_ewops_a() {
+template <typename T, size_t SROWS, size_t SCOLS> void test_ewops_a() {
+  std::cout << "Test A " << SROWS << "x" << SCOLS << "\n";
```

Comment:
do we need these messages?

Reply:
Yes, these messages are helpful to see, which case is failed and also if test is running too much times. They don't consume a lot of space, they are small and concise. I also did not see visible impact on test performance because of these messages.